### PR TITLE
Adds failure message for cyborgs attempting to check alerts while dead

### PR DESCRIPTION
**Pull request by RemieRichards**

### CODE REVIEW

**Comment by coiax** [comment]
THIS IS A FIX GUYS

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -286,6 +286,7 @@
 	set category = "Robot Commands"
 	set name = "Show Alerts"
 	if(usr.stat == DEAD)
+		src << "<span class='userdanger'>Alert: You are dead.</span>"
```
Comment (Incoming5643) [inline code/modules/mob/living/silicon/robot/robot.dm]: What a shame

Comment (coiax) [inline code/modules/mob/living/silicon/robot/robot.dm]: RIP cyborg

**Comment by xxalpha** [comment]
:cry:

**Comment by octareenroon91** [comment]
that's dead not crit

**Pull request merged**